HyeRim/bronze4_2440(별찍기3).js

```diff
@@ -0,0 +1,6 @@
+const filePath = process.platform === 'linux' ? 0 : './input.txt'
+const input = require('fs').readFileSync(filePath).toString();
+
+for(let i = input; i >= 1; i -= 1) {
+  console.log('*'.repeat(i))
+}
```

Comment:
for 문 안에서 console.log 함수를 여러번 호출하는 것보다
문자열 데이터를 만들어서 정답을 저장하고
반복문이 끝났을 때 console.log 한 번 호출하는 게 속도적으로 좋아요
```suggestion
let ans =""
for(let i = input; i >= 1; i -= 1) {
  ans += `${'*'.repeat(i)}\n`
}
console.log(ans)
```